rpc-jobs/RPC-AIO.yml

```diff
@@ -1,82 +1,135 @@
-- job:
-    name: RPC_AIO
+- defaults:
+    name: "global"
+    IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    DEPLOY_CEPH: "no"
+    DEPLOY_ELK: "yes"
+    CRON: "H H(9-21) * * 1-5"
+
+- project:
+    name: 'RPC-AIO-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - liberty:
+          branch: liberty-12.2
+          branches: "liberty-.*"
+      - mitaka:
+          branch: mitaka-13.1
+          branches: "mitaka-.*"
+      - newton:
+          branch: newton-14.0
+          branches: "newton-.*"
+      - master:
+          branch: master
+          branches: "master"
+    context:
+      - swift
+      - ceph:
+          DEPLOY_CEPH: "yes"
+          # Nothing uses USER_VARS atm
+          USER_VARS: |
```

Comment:
I think USER_VARS needs a param to ensure it gets passed though to the environment.